src/test/java/com/cdancy/jenkins/rest/features/JobsApiMockTest.java

```diff
@@ -536,6 +539,64 @@ public void testBuildJobWithParamsNonExistentJob() throws Exception {
         }
     }
 
+    public void testGetParams() throws Exception {
+        MockWebServer server = mockWebServer();
+
+        String body = payloadFromResource("/build-info.json");
+        server.enqueue(new MockResponse().setBody(body).setResponseCode(200));
+        JenkinsApi jenkinsApi = api(server.getUrl("/"));
+        JobsApi api = jenkinsApi.jobsApi();
+        try {
+            List<Parameter> output = api.buildInfo(null,"fish", 10).actions().get(0).parameters();
+            assertNotNull(output);
+            assertTrue(output.get(0).name().equals("bear"));
+            assertTrue(output.get(0).value().equals("true"));
+            assertTrue(output.get(1).name().equals("fish"));
+            assertTrue(output.get(1).value().equals("salmon"));
```

Comment:
Although unlikely to break, there should be a test for an empty parameter value. Maybe Jenkins returns null instead of an empty space.

Reply:
Agreed to all of the above.